### PR TITLE
ENT-6566/master: Fixed OOTB permissions for node modules in hub rpm

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: ENT-6566
Changelog: None

### CODE REVIEW

**Comment by nickanderson** [comment]
@craigcomstock @olehermanse this might be what we want for suppressing noise at least from RPM hub package.

File: packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in [hunk 1]
```diff
@@ -316,6 +316,9 @@ exit 0
 %prefix/httpd/conf
 %prefix/httpd/error
 %prefix/httpd/htdocs
+%prefix/httpd/htdocs/public/scripts/node_modules/highcharts/es-modules/Core/Interaction.js
+%prefix/httpd/htdocs/public/scripts/node_modules/highcharts/es-modules/highcharts.src.js
+%prefix/httpd/htdocs/public/scripts/node_modules/highcharts/es-modules/Accessibility/Accessibility.js
```
Comment (olehermanse) [inline packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in:321]: Arent't there like a thousand more .js files like this? Would it work to run a `chmod` command after `npm instal`?

Comment (nickanderson) [reply packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in:321]: I don't think so. I think you have to specify it in the spec file and the debian control file.

@vpodzime ?

Comment (vpodzime) [reply packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in:321]: The permissions are preserved if there is no`%defattr()` or `%attr()` for the listed files/directories. So yes, running `chmod` should work.

**Pull request closed without merging**